snapcraft/tests/test_parser.py

```diff
@@ -408,3 +408,46 @@ def test_n_documents(
         main(['--debug', '--index', TEST_OUTPUT_PATH])
 
         self.assertEqual(2, _get_part_list_count())
+
+    @mock.patch('snapcraft.internal.parser._get_origin_data')
+    @mock.patch('snapcraft.internal.sources.get')
+    def test_maintaner_and_description_are_included(
```

Comment:
I prefer when we have one thing per test. If you use an "and" in the test name, that's a clear indication that you have two tests. However, it might need too much duplication, or make the tests less clear if we move things to setup or functions. What do you think? Is there a nice way to split it in two?